src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import './operatorsHbarChart.scss'
+
+const colorsByCompannies: { [index: string]: string } = {
+  'אגד תעבורה': 'rgb(47, 146, 80)',
```

Comment:
let's have all colors as either `rgb` or `#hex` to align

Reply:
done